### PR TITLE
Add package docs

### PR DESCRIPTION
**Pull request by daniel-fanjul-alcuten**

### CODE REVIEW

File: cache/mock/cache.go [hunk 1]
```diff
@@ -1,14 +1,13 @@
 package heptane
 
 import (
-	"bytes"
 	"fmt"
 
-	"github.com/heptanes/heptane"
+	c "github.com/heptanes/heptane/cache"
```
Comment (manuelbcd) [inline cache/mock/cache.go]: should a package alias be more descriptive?

File: table.go [hunk 2]
```diff
@@ -1,75 +1,181 @@
...
+	fvn := make(r.FieldValuesByName, len(t.PrimaryKey)+len(t.Values))
+	for _, fn := range t.PrimaryKey {
+		fvn[fn] = kv[fn]
+	}
+	for i, fn := range t.Values {
+		if i >= len(cv) {
+			break
+		}
+		v, err := unmarshalField(t, fn, cv[i])
+		if err != nil {
```
Comment (manuelbcd) [inline table.go]: Maybe "else if" would be more efficient here

File: cache/mock/cache.go (hunk 1 above)
Comment (daniel-fanjul-alcuten) [reply cache/mock/cache.go]: which one do you suggest?

File: table.go (hunk 2 above)
Comment (daniel-fanjul-alcuten) [reply table.go]: Maybe, but I don't see it like a choice between two alternatives, but two diffferent checks that must be done.

**Pull request merged**